packages/primereact/package.json

```
@@ -0,0 +1,125 @@
+  "type": "module",
+  "exports": {
+    ".": {
+      "types": "./lib/index.d.ts",
+      "require": "./dist/index.js",
+      "import": "./lib/index.js"
+    },
+    "./lib": {
+      "types": "./lib/index.d.ts",
+      "require": "./dist/index.js",
+      "import": "./lib/index.js"
+    },
+    "./lib/*.js": {
+      "types": "./lib/*.d.ts",
+      "require": "./dist/*.js",
+      "import": "./lib/*.js"
+    },
+    "./dist": {
+      "types": "./lib/index.d.ts",
+      "require": "./dist/index.js",
+      "import": "./lib/index.js"
+    },
+    "./dist/*.js": {
+      "types": "./lib/*.d.ts",
+      "require": "./dist/*.js",
+      "import": "./lib/*.js"
+    }
+  },
+  "scripts": {
+    "build:ts": "tsc -b tsconfig.build.json && tsc-alias -p tsconfig.build.json",
+    "build:cjs": "esbuild ./src/index.ts --bundle --outfile=dist/index.js --sourcemap --packages=external --format=cjs",
+    "build:esm": "esbuild ./src/index.ts --bundle --outfile=dist/primereact.esm.js --sourcemap --packages=external --format=esm",
+    "build:umd": "rollup dist/primereact.esm.js --format=umd --file=dist/primereact.umd.js --name=@rjsf/primereact",
+    "build": "npm run build:ts && npm run build:cjs && npm run build:esm && npm run build:umd",
+    "cs-check": "prettier -l \"{src,test}/**/*.ts?(x)\"",
+    "cs-format": "prettier \"{src,test}/**/*.ts?(x)\" --write",
+    "lint": "eslint src test",
+    "precommit": "lint-staged",
+    "test": "jest",
+    "test:update": "jest --u"
+  },
+  "lint-staged": {
+    "{src,test}/**/*.ts?(x)": [
+      "eslint --fix"
+    ]
+  },
+  "files": [
+    "dist",
+    "lib",
+    "src"
+  ],
+  "engineStrict": false,
+  "engines": {
+    "node": ">=20"
+  },
+  "peerDependencies": {
+    "@rjsf/core": "^6.x",
+    "@rjsf/utils": "^6.x",
+    "primeicons": ">=6.0.0",
+    "primereact": ">=8.0.0",
+    "react": ">=18"
+  },
+  "devDependencies": {
+    "@babel/cli": "^7.23.9",
+    "@babel/core": "^7.23.9",
+    "@babel/eslint-parser": "^7.23.10",
+    "@babel/plugin-proposal-class-properties": "^7.18.6",
+    "@babel/plugin-proposal-optional-chaining": "^7.21.0",
+    "@babel/preset-env": "^7.23.9",
+    "@babel/preset-react": "^7.23.3",
+    "@babel/preset-typescript": "^7.23.3",
+    "@rjsf/core": "^6.0.0-alpha.0",
+    "@rjsf/snapshot-tests": "^6.0.0-alpha.0",
+    "@rjsf/utils": "^6.0.0-alpha.0",
+    "@rjsf/validator-ajv8": "^6.0.0-alpha.0",
+    "@rollup/plugin-replace": "^5.0.5",
+    "@types/jest": "^29.5.12",
+    "@types/react": "^18.2.58",
+    "@types/react-dom": "^18.2.19",
+    "@types/react-test-renderer": "^18.0.7",
+    "babel-jest": "^29.7.0",
+    "eslint": "^8.56.0",
+    "jest": "^29.7.0",
+    "jest-environment-jsdom": "^29.7.0",
+    "primeicons": "^7.0.0",
+    "primeflex": "^3.3.1",
+    "primereact": "^10.9.2",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-test-renderer": "^18.2.0",
+    "rimraf": "^5.0.5",
+    "rollup": "^3.29.4",
+    "typescript": "^4.9.5"
+  },
```

I just merged some cleanup changes to the `rjsf-v6` branch which simplifies the dev dependencies.
```suggestion
    "@rjsf/core": "^6.0.0-alpha.0",
    "@rjsf/snapshot-tests": "^6.0.0-alpha.0",
    "@rjsf/utils": "^6.0.0-alpha.0",
    "@rjsf/validator-ajv8": "^6.0.0-alpha.0",
    "@rollup/plugin-replace": "^5.0.5",
    "eslint": "^8.56.0",
    "primeicons": "^7.0.0",
    "primeflex": "^3.3.1",
    "primereact": "^10.9.2"
  },
```